src/SayMore/UI/ProjectWindow/ProjectWindow.cs

```diff
@@ -61,9 +63,51 @@ private ProjectWindow()
 			ExceptionHandler.AddDelegate(AudioUtils.HandleGlobalNAudioException);
 
 			InitializeComponent();
+			// ReSharper disable once VirtualMemberCallInConstructor
 			_titleFmt = Text;
 			_menuShowMPlayerDebugWindow.Tag = _menuProject.DropDownItems.IndexOf(_menuShowMPlayerDebugWindow);
 			_menuProject.DropDownItems.Remove(_menuShowMPlayerDebugWindow);
+
+			KeyPreview = true;
+			KeyDown += Project_KeyDown;
```

Comment:
I think you can just override OnKeyDown, rather than hooking up a handler. Better yet, I think you could leave KeyPreview false and just override ProcessDialogKey or ProcessCmdKey. (According to [this](https://stackoverflow.com/questions/2386695/disadvantage-of-setting-form-keypreview-true/2386844#2386844), KeyPreview is a bit of an anachronism held over from the days of VB6.)